PracticeMode.acs.cs

```diff
@@ -0,0 +1,4 @@
+editActionMap("playMap.sae");

```

Comment:
could you remove this file from the pull request? this looks like a local file for people

Reply:
yes can we create a new folder somewhere for client side scripts perhaps?